code/game/jobs/job/science.dm

```diff
@@ -57,8 +57,7 @@
 	economic_modifier = 7
 	access = list(access_robotics, access_tox, access_tox_storage, access_research, access_xenobiology, access_xenoarch)
 	minimal_access = list(access_tox, access_tox_storage, access_research, access_xenoarch)
-	alt_titles = list("Xenoarcheologist", "Anomalist", "Phoron Researcher")
-
+	alt_titles = list("Xenoarcheologist", "Anomalist", "Phoron Researcher", "Geologist", "Physicist")

```

Comment:
Still don't see exactly what those last two are for.

Reply:
For fun. It encourages science RP merely by the titles existing.